docs/reference/aggregations/pipeline/change-point-aggregation.asciidoc

```diff
@@ -0,0 +1,99 @@
+[role="xpack"]
+[[search-aggregations-change-point-aggregation]]
+=== Change Point aggregation
+++++
+<titleabbrev>Change Point</titleabbrev>
+++++
+
+experimental::[]
+
+A sibling pipeline that detects a change point in a metric. Given a distribution of values provided by the sibling
+multi-bucket aggregation, this aggregation where find the bucket at which the distribution of points significantly
+changes.
+
+
+
+[[change-point-agg-syntax]]
+==== Parameters
+
+`buckets_path`::
+(Required, string)
+Path to the buckets that contain one set of values in which to detect a change point. There must be at least 21 bucketed
+values. Fewer than 1,000 is preferred.
+For syntax, see <<buckets-path-syntax>>.
+
+==== Syntax
+
+A `change_point` aggregation looks like this in isolation:
+
+[source,js]
+--------------------------------------------------
+{
+  "change_point": {
+    "buckets_path": "date_histogram>_count" <1>
+  }
+}
+--------------------------------------------------
+// NOTCONSOLE
+<1> The buckets containing the values to test against.
+
+[[avg-bucket-agg-response]]
```

Comment:
```suggestion
[[change-point-agg-response]]
```

I think this duplicated anchor ID is causing the following docs build error:
> 12:15:20 INFO:build_docs:asciidoctor: WARNING: aggregations/pipeline/change-point-aggregation.asciidoc: line 41: id assigned to section already in use: avg-bucket-agg-response

Reply:
lulz, copy paste FAIL!